Frontend/src/components/BoardDetails.jsx

```diff
@@ -0,0 +1,59 @@
+import { useParams } from 'react-router-dom';
+import { useState, useEffect } from 'react'
+import "./BoardDetails.css";
+import BoardCard from './BoardCard';
+
+const LOGO = "https://i.vimeocdn.com/video/557834687-b8d55eb049d1702b589b4ad62c31fe18ac0f44f0316546d6aef20c61be70435c-d_640?f=webp";
+
+function BoardDetails () {
+    const {boardId} = useParams();
+    const [cards, setCards] = useState([])
+
+    useEffect(() => {
+        fetch(`http://localhost:3000/boards/${boardId}`)
```

Comment:
should be able to move this `http://localhost:3000/boards/` to a const as well for reuse